config/client.lua

```diff
@@ -1,16 +1,16 @@
 return {
     spawns = {
         {
-            label = 'Legion Square',
+            label = 'legion_square',
             coords = vec4(195.17, -933.77, 29.7, 144.5)
         },
         {
-            label = 'Paleto Bay',
+            label = 'paleto_bay',
             coords = vec4(80.35, 6424.12, 31.67, 45.5),
         },
         {
-            label = 'Motels',
+            label = 'motels',
```

Comment:
Why these changes?

Reply:
not in every language these places have the same names

Reply:
Configs shouldn't hold locales, they should hold values that you can change only. These ones are not meant to be changed so they are not meant for the config.

Reply:
In this case, I don't think locales are appropriate as this is a highly configurable and dynamic array. If someone wants to add a new location they will then need to add it to the locales files as well.

Reply:
What do you think about such a scenario?
en.json
```json
{
    "last_location": "Last Location",
}
```
pl.json
```json
{
    "last_location": "Ostatnia pozycja",
    "Legion Square": "Kwadraciak",
    "Paleto Bay": "Paleto",
    "Motels": "Motele"
}
```

Reply:
Are you trying to support client side locales? If not, I think the least amount of work for server owners will just be to edit these in config only and not bother with the locales file.

Reply:
Exactly